samri/utilities.py

```diff
@@ -67,7 +67,7 @@ def bids_autofind(bids_dir,
 			raise ValueError("The reconstructed file path based on the substitution dictionary and the path template, is not identical to the corresponding path, found by `nipype.interfaces.io.DataFinder`. See string values above.")
 		substitutions.append(substitution)
 
-	return substitutions
+	return path_template, substitutions
```

Comment:
I don't see the path template getting changed in this function - why would you return it?

Reply:
I saw function calls of this function which expected template and substitutions